main.py

```diff
@@ -0,0 +1,54 @@
+from fastapi import Depends, FastAPI, HTTPException
+import schemas
+from db_operations import crud
+from sqlalchemy.orm import Session
+from db_operations.database import SessionLocal

```

Comment:
Imports should be grouped and separated by a blank line according to standard library, external libraries, and custom imports. This practice enhances both code readability and maintainability. 
Read more about topic [here](https://peps.python.org/pep-0008/#imports)